tutorials/remote-staking-xcm.md

```diff
@@ -0,0 +1,118 @@
+
+!!! note
+    Astute readers may notice the selected account below is named “Academy.” It does not matter which account you have selected in Moonbase Alpha Polkadot.js Apps. This is because you're not submitting the prepared transaction, only copying the encoded call data, which does not contain a reference to the sending account. 
+
+![Moonbase Alpha Polkadot JS Apps Extrinsics Page](/images/tutorials/remote-staking-via-xcm/xcm-stake-3.png)
+
+## Sending the XCM Instructions from the Moonbase Relay chain
+
+In another tab, head to [Moonbase Relay Polkadot.Js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/extrinsics){target=_blank}. Click on the **Developer** tab and press **Extrinsics**. 
+
+![Moonbase Relay Polkadot JS Apps Home](/images/tutorials/remote-staking-via-xcm/xcm-stake-4.png)
+
+### Building the Destination Multilocation 
+Let’s get started crafting our XCM message that will transport our remote execution instructions to the Moonbase Alpha Parachain to ultimately stake our desired amount of DEV tokens to a chosen collator. To get started, take the following steps: 
+
+1. Unlike the prior steps where the selected account wasn’t relevant, the account selected here must be the account associated with your derivative account 
+2. Select the `xcmPallet`
+3. Select the `send` function
+4. Select version `v1`
+5. Select a parents value of `0`. Recall we’re creating the multilocation to identify the destination of our XCM message. In this case, we’re sending the message from the relay chain to the Moonbase alpha parachain. As we’re already at the relay chain level, we don’t need to “hop up” any levels so we’ll enter a parents value of `0`.
+6. We need one piece of information, namely, a parachain id to describe our destination of Moonbase Alpha, so enter `X1` here
+7. Select `parachain ID` 
+8. Enter `1000`, the parachain ID of Moonbase Alpha 
+
+![Moonbase Relay Polkadot JS Apps Extrinsics Page](/images/tutorials/remote-staking-via-xcm/xcm-stake-5.png)
+
+In the next section, we’ll start assembling the XCM instructions. 
+
+### Preparing the Structure of the XCM Message
+
+1. Select `V2` for XcmVersionedXcm
+2. Our XCM Message is going to have 3 distinct XCM instructions, so press the first **Add Item** button 3 times. 
+3. Below the first XCM Instruction of `Withdraw Asset`, we need to add the asset we’re going to withdraw here, so press the **Add Item** button below `Withdraw Asset` once. 
+
+![Preparing the structure of the XCM message](/images/tutorials/remote-staking-via-xcm/xcm-stake-7.png)
+
+### Assembling the Contents of the XCM Message
+
+Now we’re ready for the fun part! To construct the XCM message that will remotely stake funds on the Moonbase Alpha parachain, take the following steps:
+
+1. Double check that `V2` is selected as the XCM version
```

Comment:
Normally I would collapse a lot of the multilocation or multiple settings into a single dropdown with just 1 number, check here: https://docs.moonbeam.network/builders/xcm/remote-evm-calls/#build-xcm-remote-evm